core/src/main/java/com/linecorp/armeria/common/stream/FilteredStreamMessage.java

```diff
@@ -191,9 +204,6 @@ public void onSubscribe(Subscription s) {
         @Override
         public void onNext(T o) {
             ReferenceCountUtil.touch(o);
-            if (!filterSupportsPooledObjects) {
```

Comment:
@anuraaga I found out that the object is always `false` when `subscribedWithPooledObjects` is `false` even though `filterSupportsPooledObjects` is `true`. So I changed to use `filterSupportsPooledObjects` when subcribing the `delegate`. PTAL 🙇

Reply:
Ah this seems more natural, thanks